### DISCUSSION TITLE
Wrong Spelling of author's username is being fetched in the Author(s): section

### DISCUSSION THREAD

**Discussion by Parth0x108** [solodit/feedback#53] [New Feature Requests]
In the "Author(s)" section of this finding, there's a mistake in one of the author names. Instead of the correct username, which should be `ParthMandale` (Sherlock username), it has been incorrectly listed as `ParthMale`

In the "Found by" section, the username `ParthMandale` is accurately represented. However, there's a discrepancy in the "Author(s):" section, where the username is incorrect.

Please correct the spelling of the username in the "Author(s)" section.

Link of the issue - 
https://solodit.xyz/issues/m-1-user-can-vote-even-when-they-have-0-locked-mento-edge-case-sherlock-mento-git

**Comment by wang6131** [comment]
@Parthmandale 
Thank you for your report.
Corrected the spelling of your username.
